### ISSUE TITLE
[Content styleguide] Content doesn’t align with style guidance in VA.gov content style guide. (05.08.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#55853]
#### VFS team name
VA Online Scheduling (VAOS)

#### VFS product name
VA Online Scheduling

#### VFS feature name
Appointment List

#### Point of Contact/Reviewers
Cheryl Evans - @CherylEvans - Content

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Content doesn’t align with style guidance in VA.gov content style guide.

### Issue Details
Home > Health Care > Schedule and Manage health appointments > Your appointments > Days of the week are abbreviated unnecessarily.  Abbreviated days of the week should have a period at the end of the abbreviation > Abbreviate days and months when space is very limited, such as in callout boxes, alerts, or promo components, etc.
Mon., Tue., Wed., Thu., Fri., Sat., Sun.

### Link, screenshot or steps to recreate
https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/42025_Cheryl_1_%202023-03-27%20at%2012.28.55%20PM.png  https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images

### VA.gov Experience Standard
[Category Number 05, Issue Number 08](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

### Platform Recommendation
I recommend not using abbreviations unless there is a space constraint.  If abbreviations must be used, follow the guidelines for correct usage > https://design.va.gov/content-style-guide/dates-and-numbers

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by ldelacosta** [comment]
@shiragoodman - we will review this post launch of the Appointment List Redesign feature. We have created a [ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/57148) for our team. cc: @JRRoof

**Comment by shiragoodman** [comment]
thanks @ldelacosta . Per our guidance above, we request that this ticket remain open until the issue is resolved. Thanks for your understanding.
